src/index.ts

```diff
@@ -139,6 +132,7 @@ export const eventHandler = async (
 
 export default async function (manager: Manager, settings: ComponentSettings) {
   manager.addEventListener('pageview', event => {
+    conversionLinkerHandler(event, settings)
```

Comment:
we don't need this check anymore? `if (client.url.searchParams.get('gclid')) {`

Reply:
I think it is redundent because the conversionLinket.ts is already checking it:
https://github.com/managed-components/google-ads/blob/cdf94ebf352d7c254d626f0b61d775cbde507e6d/src/conversionLinker.ts#L10

Reply:
should we put this one too under gclid check? https://github.com/managed-components/google-ads/blob/cdf94ebf352d7c254d626f0b61d775cbde507e6d/src/conversionLinker.ts#L16

Reply:
I did not touch this function because it is not part of the scope of the issue we're trying to handle -- the collection of gclid automaticaly by the MC. 

I _believe_  (again, I did not fully delved into this) that this is supposed to get the gclid value from a cookie and append the `_gl` query parameters to links to domains that are passed under a setting. As it is a very different thing, I did not touch it. If it should enter the scope lemme know!